integration_tests/pages/assess/clarificationNoteConfirmPage.ts

```diff
@@ -11,11 +11,11 @@ export default class SufficientInformationPage extends Page {
     return cy.get('a').contains('Return to dashboard').click()
   }
 
-  confirmUserDetails(user: User) {
+  confirmUserDetails(data: AnyValue) {
```

Comment:
I think it would be better to pass the application in here and use `retrieveQuestionResponseFromFormArtifact` to get the user details